google/cloud/bigquery/client.py

```diff
@@ -2757,7 +2759,12 @@ def load_table_from_json(
 
         job_config.source_format = job.SourceFormat.NEWLINE_DELIMITED_JSON
 
-        if job_config.schema is None:
+        # check whether the table already exists with a schema
+        # if it does already have a schema, we will not tell the `backend to use autodetect.
+        table = _table_arg_to_table(destination, default_project=self.project)
```

Comment:
This method doesn't make an API call to check whether the table exists, which is what we need to do. This method is just creating a local object representing the table. You want to reference the docs that demonstrate how to check whether a table exists in order to find the correct method to use.